help/invite-new-users.md

```diff
@@ -42,9 +45,10 @@ permission to invite users.
 1. Click **Invite**.
 
 !!! warn ""
-    **Note**: As an anti-spam measure, the number of email invites you can send in a day is
-    limited on the Zulip Cloud Free plan. [Contact support](/help/contact-support)
-    if you hit the limit and want to invite more users.
+    **Note**: As an anti-spam measure, the number of email invitations
+    you can send in a day is limited on the Zulip Cloud Free plan.
+    [Contact support](/help/contact-support) if you hit the limit and
+    want to invite more users.
```

Comment:
It might be a good idea to mention generating an invitation link and sending the email yourself as a workaround here; it might save some stress and support tickets for people where that workaround is totally sufficient for their needs.